tikv-client/src/main/java/com/pingcap/tikv/expression/visitor/ProtoConverter.java

```diff
@@ -45,6 +45,7 @@
           .put(TimestampType.class, "Time")
           .put(BytesType.class, "String")
           .put(StringType.class, "String")
+          .put(TimeType.class, "Duration")
```

Comment:
Should its name be Time or Duration? I can see them both.

Reply:
It is duartion. Time is for date, datetime, and timestamp.